packages/contracts/README.md

```diff
@@ -160,5 +160,23 @@ If you are using a network which Etherscan supports you can verify your contract
 npx hardhat etherscan-verify --api-key ... --network ...
 ```
 
+### Other hardhat tasks
+
+To whitelist deployers on Mainnet you must have the whitelist Owner wallet connected, then run:
+```bash
+npx hardhat whitelist \
+  --use-ledger true \  # `network` MUST equal your env var `CONTRACTS_TARGET_NETWORK`
```

Comment:
I didn't set the `CONTRACTS_TARGET_NETWORK` and was able to run this without a problem